src/components/jars/Jar.tsx

```diff
@@ -21,9 +21,21 @@ export type JarProps = {
 }
 
 export type SelectableJarProps = JarProps & {
+  tooltipText?: string
+  isOpen?: boolean
+  isSelectable: boolean
+  isSelected: boolean
+  variant?: 'default' | 'warning'
+  onClick: (index: JarIndex) => void
+}
+type showingUTXOS = { index: String; show: boolean }
+export type SelectableSendJarProps = JarProps & {
+  tooltipText: string
   isSelectable: boolean
   isSelected: boolean
   variant?: 'default' | 'warning'
+  showingUTXOS: showingUTXOS
+  setshowingUTXOS: (val: showingUTXOS) => void
   onClick: (index: JarIndex) => void
 }
```

Comment:
Since this is nearly the same as `SelectableJarProps`, can it be simplified to
```typescript
export type SelectableSendJarProps =  SelectableJarProps & ShowingUtxos
```
?

Also, instead of mandating a value for property of type `ShowingUtxos` and later use a quirk by setting
```
setShowUtxos({
        jarIndex: '',
        isOpen: false,
      })
```

can it be optional (and set it to `undefined` if not used)?

Reply:
left-over removed and about setShowUtxos Yes.it's better if we set it to undefined rather than keeping it empty